packages/material-ui-codemod/README.md

```diff
@@ -0,0 +1,41 @@
+# Material-UI codemod
+
+This repository contains a collection of codemod scripts based for use with
+[JSCodeshift](https://github.com/facebook/jscodeshift) that help update Material-UI
+APIs.
+
+## Setup & Run
+
+- `npm install -g jscodeshift`
+- `npm install material-ui-codemod`
+- `jscodeshift -t <codemod-script> <file>`
```

Comment:
That sounds like a good idea to change to `<path>`.
Let's see what facebook think about it https://github.com/reactjs/react-codemod/pull/37 :grin:.